etc/rc1

```diff
@@ -18,6 +18,9 @@ flux module load -r 0 userdb ${FLUX_USERDB_OPTIONS}
 # make flux wreckrun/submit -onokz the default (override w/ -o kz):
 flux wreck setopt nokz
 
+flux module load -r all job-ingest
+flux module load -r 0 job-manager
+
```

Comment:
Just making sure that adding the job-ingest and job-manager modules to the default `rc1` script was meant to be part of this PR? (If so that is something to note for release notes...)

Reply:
Yes it was intended.  I'll amend the description.